legend-engine-xt-serviceStore-grammar/pom.xml

```diff
@@ -121,6 +121,10 @@
             <groupId>org.finos.legend.engine</groupId>
             <artifactId>legend-engine-xt-serviceStore-pure</artifactId>
         </dependency>
+        <dependency>
```

Comment:
why this change?

Reply:
required while compiling serviceStoreConnection, which contains authentication specs

Reply:
just for cleanup. There is specific section where we have all the pure dependencies together, so moved this dependency also there